### DISCUSSION TITLE
Typhon 1 jail the ventilation does not work

### DISCUSSION THREAD

**Discussion by Purpursarkans** [FakeFishGames/Barotrauma#15439] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When the door is closed, prisoners begin to suffocate, although there is ventilation inside the cell, which means there should be an influx of oxygen

### Reproduction steps

Lock jail on Typhon 1

https://github.com/user-attachments/assets/194de97d-39c8-494d-a2bc-b61d8057e779

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Linux

**Comment by SomeRandomNoobKekeke** [comment]
Ye, it should sustain 3 people because this thing for Typhon is 3, currently it can sustain only 2

https://github.com/FakeFishGames/Barotrauma/blob/38ca721cb0fd94aadbb9c1886ba2338bbd5f1180/Barotrauma/BarotraumaShared/SharedSource/Events/Missions/EscortMission.cs#L85

**Comment by Regalis11** [comment]
Addressed in https://github.com/FakeFishGames/Barotrauma-development/commit/01cc82e4754c85acf97e7749dee50b19e9975009 (commit in our private repo), fix coming up in the next unstable build and the upcoming release.
